<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customWorkingShiftsWidget</class>
 <widget class="QWidget" name="customWorkingShiftsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background-color: #dfdfdf;
	font: 600 12pt &quot;Segoe UI&quot;;
	color: #324b4c;
}

QMenu {
	background-color: #d5d5d5;
	border-radius: 5px;
}
QMenu::item {
	background-color: transparent;
	font: 600 10pt &quot;Segoe UI&quot;;
	color: #324b4c;
	padding: 5px;
}
QMenu::item:selected {
	background-color: #c1c4c9;
}

QScrollBar:vertical {
	width: 20px;
	background: none;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
	background: #dfdfdf;
}

QPushButton {
	background-color: #c9c9c9;
	border: none;
	border-radius: 10px;
	padding: 5px;
	font-size: 12pt;
}

QPushButton:hover{
	background-color: #aeaeae;
}

QLineEdit {
	border: none;
	font-size: 11pt;
	border-bottom: 1px solid #7c9399;
}

#modelHolder QLabel, #newModelInfoHolder * , #widget_11 *{
	font-size: 10pt;
}

#optionsHolder, #optionsHolder_2 {
	border-bottom: 2px dashed #7c9399;
}

#pageTitle {
	font: 700 13pt &quot;Segoe UI&quot;;
}

#userIcon, #calendarBtn {
	background-color: #dfdfdf;
}

#usernameLabel {
	font-size: 11pt;
}

#workerPlaceLineEdit, #workerPositionLineEdit {
	font-weight: 700;
}

#widget_3 * {
	font-size: 11pt;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item alignment="Qt::AlignTop">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="headerHolder" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignLeft">
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="pageTitle">
           <property name="text">
            <string>РАБОТНИ СМЕНИ</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QWidget" name="userHolder" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="userIcon">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../../resources.qrc">
                <normaloff>:/icons/app/assets/icons/User--Streamline-Feather.svg</normaloff>:/icons/app/assets/icons/User--Streamline-Feather.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="usernameLabel">
              <property name="text">
               <string>admin</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QWidget" name="workerInfoHolder" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>35</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item alignment="Qt::AlignTop">
          <widget class="QWidget" name="workerShiftsHolder" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_11" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>5</number>
               </property>
               <item>
                <layout class="QGridLayout" name="gridLayout"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="newModelInfoHolder" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QWidget" name="widget_6" native="true">
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <item>
                   <layout class="QGridLayout" name="newModelLayout">
                    <property name="spacing">
                     <number>10</number>
                    </property>
                    <item row="0" column="6">
                     <widget class="QLabel" name="label_21">
                      <property name="text">
                       <string>Общо:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="5">
                     <widget class="QWidget" name="shiftEndWidget" native="true">
                      <layout class="QHBoxLayout" name="horizontalLayout_9">
                       <property name="spacing">
                        <number>5</number>
                       </property>
                       <property name="leftMargin">
                        <number>0</number>
                       </property>
                       <property name="topMargin">
                        <number>0</number>
                       </property>
                       <property name="rightMargin">
                        <number>0</number>
                       </property>
                       <property name="bottomMargin">
                        <number>0</number>
                       </property>
                       <item>
                        <widget class="QTimeEdit" name="shiftEnd">
                         <property name="time">
                          <time>
                           <hour>17</hour>
                           <minute>0</minute>
                           <second>0</second>
                          </time>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_15">
                         <property name="text">
                          <string>ч.</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item row="0" column="4">
                     <widget class="QLabel" name="label_10">
                      <property name="text">
                       <string>Край:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="5" column="3">
                     <widget class="QWidget" name="hourlyStartWidget" native="true">
                      <layout class="QHBoxLayout" name="horizontalLayout_10">
                       <property name="spacing">
                        <number>5</number>
                       </property>
                       <property name="leftMargin">
                        <number>0</number>
                       </property>
                       <property name="topMargin">
                        <number>0</number>
                       </property>
                       <property name="rightMargin">
                        <number>0</number>
                       </property>
                       <property name="bottomMargin">
                        <number>0</number>
                       </property>
                      </layout>
                     </widget>
                    </item>
                    <item row="0" column="3">
                     <widget class="QWidget" name="shiftStartWidget" native="true">
                      <layout class="QHBoxLayout" name="horizontalLayout_8">
                       <property name="spacing">
                        <number>5</number>
                       </property>
                       <property name="leftMargin">
                        <number>0</number>
                       </property>
                       <property name="topMargin">
                        <number>0</number>
                       </property>
                       <property name="rightMargin">
                        <number>0</number>
                       </property>
                       <property name="bottomMargin">
                        <number>0</number>
                       </property>
                       <item>
                        <widget class="QTimeEdit" name="shiftStart">
                         <property name="calendarPopup">
                          <bool>false</bool>
                         </property>
                         <property name="time">
                          <time>
                           <hour>8</hour>
                           <minute>0</minute>
                           <second>0</second>
                          </time>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_14">
                         <property name="text">
                          <string>ч.</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_5">
                      <property name="text">
                       <string>Смяна:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="7">
                     <widget class="QLabel" name="shiftTotalMins">
                      <property name="text">
                       <string>480</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="8">
                     <widget class="QLabel" name="label_27">
                      <property name="text">
                       <string>мин.</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="2">
                     <widget class="QLabel" name="label_7">
                      <property name="text">
                       <string>Начало:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QLabel" name="label">
                      <property name="text">
                       <string>Покивка:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="1" alignment="Qt::AlignLeft">
                     <widget class="QWidget" name="widget_3" native="true">
                      <layout class="QHBoxLayout" name="horizontalLayout_6">
                       <property name="spacing">
                        <number>5</number>
                       </property>
                       <property name="leftMargin">
                        <number>0</number>
                       </property>
                       <property name="topMargin">
                        <number>0</number>
                       </property>
                       <property name="rightMargin">
                        <number>0</number>
                       </property>
                       <property name="bottomMargin">
                        <number>0</number>
                       </property>
                       <item alignment="Qt::AlignLeft">
                        <widget class="QLineEdit" name="shiftBreakLineEdit">
                         <property name="maximumSize">
                          <size>
                           <width>45</width>
                           <height>16777215</height>
                          </size>
                         </property>
                         <property name="placeholderText">
                          <string>60</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_2">
                         <property name="text">
                          <string>мин.</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QLineEdit" name="workingShiftsNameLineEdit"/>
                    </item>
                   </layout>
                  </item>
                  <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                   <widget class="QPushButton" name="acceptWorkingShiftsBtn">
                    <property name="focusPolicy">
                     <enum>Qt::NoFocus</enum>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">*{
	padding: 0px;
	background-color: #dfdfdf;
}</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="icon">
                     <iconset resource="../../../resources.qrc">
                      <normaloff>:/icons/app/assets/icons/Check-Square--Streamline-Solar-Broken-#008b69.svg</normaloff>:/icons/app/assets/icons/Check-Square--Streamline-Solar-Broken-#008b69.svg</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>24</width>
                      <height>24</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="operationsHolder" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QFrame" name="optionsHolder">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>10</number>
            </property>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>888</width>
        <height>563</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="workingShiftsTable" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableView" name="workingShiftsTableView">
            <property name="styleSheet">
             <string notr="true">QHeaderView:section{
	font: 700 10.5pt &quot;Segoe UI&quot;;
	background-color: #dfdfdf;
	padding-left: 10px;
	padding-top:10px;
	padding-right:10px;
}
QVerticalView:section{
	min-height: 30;
}
QAbstractItemView{
	alternate-background-color: #d3d3d3;
	font: 10.5pt &quot;Segoe UI&quot;;
	selection-background-color: #aeaeae;
	selection-color: #fefefe;
}
QAbstractItemView:item{
	selection-background-color: #aeaeae;
	selection-color: #fefefe;
}</string>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="cornerButtonEnabled">
             <bool>false</bool>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
